docs/src/prototypes/customToolbar/CustomToolbar.tsx

```diff
@@ -0,0 +1,341 @@
+        _.invoke(props, 'onFocus', args)
+      }}
+    />
+  )
+}
+
+const commonLayout: CustomToolbarLayout = props => [
+  ...((props.isRecording
+    ? [
+        // Status has focus
+        // {
+        //   key: 'recording1',
+        //   kind: 'custom',
+        //   fitted: 'horizontally',
+        //   content: <FocusableStatus state="error" title="Recording" styles={{ margin: '0 9px' }} />,
+        //   variables: { primary: true },
+        // },
+
+        // TODO: Winner. Meets visual parity with Teams and code is most sane.  Confirm acc with Juraj.
+        // Flexbox has focus
+        {
+          key: 'recording2',
+          kind: 'custom',
+          fitted: 'horizontally',
+          content: (
+            <FocusableFlexBox fill gap="gap.medium" vAlign="center">
+              <Status state="error" title="Recording" />
+            </FocusableFlexBox>
+          ),
+          variables: { primary: true },
+        },
+
+        // // Magic - status has focus, Flexbox draws focus ring
+        // {
+        //   key: 'recording3',
+        //   kind: 'custom',
+        //   fitted: 'horizontally',
+        //   content: (
+        //     <FocusableFlexBox fill gap="gap.medium" vAlign="center" data-is-focusable={false}>
+        //       <FocusableStatus
+        //         state="error"
+        //         title="Recording"
+        //         variables={{ isFromKeyboard: false }}
+        //       />
+        //     </FocusableFlexBox>
+        //   ),
+        //   variables: { primary: true },
+        // },
+      ]
+    : []) as any),
+  {
+    key: 'timer-custom',
+    kind: 'custom',
+    variables: { primary: true },
+    fitted: 'horizontally',
+    content: (
+      <FocusableFlexBox fill gap="gap.medium" vAlign="center">
+        10:45
+      </FocusableFlexBox>
+    ),
+  },
+
+  { key: 'timer-divider', kind: 'divider' },
+
+  {
+    active: props.cameraActive,
+    icon: {
+      name: props.cameraActive ? 'call-video' : 'call-video-off',
+      size: 'large',
+    },
+    key: 'camera',
+    onClick: () => _.invoke(props, 'onCameraChange', !props.cameraActive),
+    variables: { primary: true },
+  },
+
+  {
+    active: props.micActive,
+    icon: {
+      name: props.micActive ? 'mic' : 'mic-off',
+      size: 'large',
+    },
+    key: 'mic',
+    onClick: () => _.invoke(props, 'onMicChange', !props.micActive),
+    variables: { primary: true },
+  },
+
+  {
+    active: props.screenShareActive,
+    icon: {
+      name: props.screenShareActive ? 'call-control-close-tray' : 'call-control-present-new',
+      size: 'large',
+    },
+    key: 'screen-share',
+    onClick: () => _.invoke(props, 'onScreenShareChange', !props.screenShareActive),
+    variables: { primary: true },
+  },
+
+  {
+    key: 'more',
+    icon: {
+      name: 'more',
+      size: 'large',
+    },
+
+    onClick: () => {
+      _.invoke(props, 'onMoreClick')
+    },
+
+    variables: {
+      primary: true,
+    },
+  },
```

Comment:
The exampe does not show a spin button, but we will need to figure out how that is going to be transformed into a menu item for the overflow. Should we generate two menu items for it? Should we introduce something similar to the chrome zoom menu - which is a group of menu items aligned horizontaly?

![image](https://user-images.githubusercontent.com/13742664/60335182-84e49980-999d-11e9-8dd8-bd24fba3c92d.png)